### PR TITLE
Add Astra icon

### PR DESCRIPTION
**Pull request by CodeShakingSheep**
image

**Issue:** closes https://github.com/simple-icons/simple-icons/issues/11104

**Popularity metric:**

The Similarweb rank is 70,654. See https://www.similarweb.com/website/wpastra.com

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Downloaded icon from website's header, removed wordmark, resized, unified paths, optimized.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -1587,6 +1587,11 @@
             "hex": "00665E",
             "source": "https://www.astonmartin.com"
         },
+        {
+            "title": "Astra",
+            "hex": "FFC300",
+            "source": "https://cdn.shortpixel.ai/spai/ret_img/wpastra.com/wp-content/uploads/2019/05/astra-logo.svg"
```
Comment (uncenter) [inline _data/simple-icons.json:1593]: See https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines, this is incorrect.

**Review by uncenter — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (CodeShakingSheep) [reply _data/simple-icons.json]: Yeah right. Just updated the source.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -1587,6 +1587,11 @@
             "hex": "00665E",
             "source": "https://www.astonmartin.com"
         },
+        {
+            "title": "Astra",
+            "hex": "FFC300",
+            "source": "https://wpastra.com"
+        },
```
Comment (uncenter) [inline _data/simple-icons.json:1594]: This should actually probably be sourced from https://wpastra.com/brand-assets/. They have logos there, and the first listed brand color seems more accurate than the yellow: `#5c2ede`.

File: icons/astra.svg
Comment (uncenter) [inline icons/astra.svg:1]: Logo looks like a perfect match compared to the header though I'd like to make sure it matches once we update the source to their branding page.

**Review by uncenter — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 2 above)
Comment (uncenter) [reply _data/simple-icons.json:1594]: Screenshot 2024-06-07 at 10 03 56 (Finder)

You sure?

Comment (CodeShakingSheep) [reply _data/simple-icons.json]: Ah yes, I just downloaded the icons zip and there were only svgs. 
image


You're right about the logo zip though. There are svg files present. I just updated color and source.

File: icons/astra.svg
Comment (CodeShakingSheep) [reply icons/astra.svg:1]: Ok, if I need to redo the icon based on the svg from the branding page, let me know.

File: _data/simple-icons.json (hunk 2 above)
Comment (uncenter) [reply _data/simple-icons.json]: Hmm well actually none of the logos here match the outline treatment you've done here. Maybe change the source back if we want to stick with it this way... or we could change to the filled in treatment? Up to you.

Comment (CodeShakingSheep) [reply _data/simple-icons.json]: `astra_logo03.svg` from the logo zip file matches the outline version of the logo.

image

So, I think we can leave the source and icon as they are? What do you think @uncenter?

Comment (uncenter) [reply _data/simple-icons.json]: From what I saw the outline was significantly thinner then your outline? I could have been wrong though I didn't fully check.

Comment (CodeShakingSheep) [reply _data/simple-icons.json]: Really? I didn't fully check it either. Anyway, I did a version without the outline. This is how it looks like now.
image

**Review by adamrusted — APPROVED** [review]
Spot on, thanks @CodeShakingSheep!

**Pull request merged**